include/nbl/video/ILogicalDevice.h

```diff
@@ -632,6 +632,9 @@ class NBL_API2 ILogicalDevice : public core::IReferenceCounted, public IDeviceMe
             return updateDescriptorSets({pDescriptorWrites,descriptorWriteCount},{pDescriptorCopies,descriptorCopyCount});
         }
 
+        // should this be joined together with the existing updateDescriptorSets?
+        bool nullifyDescriptors(const std::span<const IGPUDescriptorSet::SDropDescriptorSet> dropDescriptors, asset::IDescriptor::E_TYPE descriptorType);
```

Comment:
no don't join, cause `updateDescriptorSets` can't write null descs and takes a different parameter struct

Making this take a `descriptorType` while allowing for a full `SDropDescriptorSet` is some seriously broken design